samples/nrf9160/http_application_update/prj.conf

```diff
@@ -0,0 +1,61 @@
+# General config
+CONFIG_NEWLIB_LIBC=y
+CONFIG_TEST_RANDOM_GENERATOR=y
+CONFIG_ASSERT=y
+CONFIG_REBOOT=y
+
+# Trusted execution
+CONFIG_TRUSTED_EXECUTION_NONSECURE=y
+
+# Network
+CONFIG_NETWORKING=y
+CONFIG_NET_SOCKETS=y
+CONFIG_NET_SOCKETS_OFFLOAD=y
+
+# LTE link control
+CONFIG_LTE_LINK_CONTROL=y
+CONFIG_LTE_AUTO_INIT_AND_CONNECT=y
+
+# BSD library
+CONFIG_BSD_LIBRARY=y
+
+# Library for buttons and LEDs
+CONFIG_DK_LIBRARY=y
+CONFIG_DK_LIBRARY_INVERT_LEDS=n
+
+# Console
+CONFIG_CONSOLE_SUBSYS=y
+CONFIG_CONSOLE_HANDLER=y
+CONFIG_CONSOLE_GETCHAR=y
+
+# Main thread
+CONFIG_MAIN_THREAD_PRIORITY=7
+
+# Heap and stacks
+CONFIG_HEAP_MEM_POOL_SIZE=16384
```

Comment:
Is such a huge heap really needed? I know we need some for `bsdlib`, but is there any other, memory-hungry user?

Reply:
This was copied from another sample that uses the http download client. Please suggest another value and I will change it.

Reply:
Well, it all depends on the sample, really. `nrf_cloud` is a heavy heap user, so samples using it enable 16K. CoAP also uses some, so we have 4K enabled. On the other hand, we have `mqtt_simple` that only uses heap for bsdlib - for this sample 1K is enough.

Looking at the code it seems that we have a similar situation as in `mqtt_simple` - I'd try with 1K then.

Reply:
Changed to 1k, tested and it works.